src/Symfony/Component/Mailer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add DSN param `retry_period` to override default email transport retry period
+ * Add DSN param `force_ipv4` to enforce binding to IPv4
```

Comment:
should this `source_ip` instead? it looks like this would be a bit more powerful, isn't it?